microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md

```diff
@@ -58,13 +58,10 @@ You'll need to take the following steps to onboard non-Windows devices:
 
 ## Offboard non-Windows devices
 
-1. Follow the third party's documentation to disconnect the third-party solution from Microsoft Defender for Endpoint.
+For macOS and Linux devices, you can choose to offboard through Microsoft Defender for Endpoint.
+ 1. In the navigation pane, select ***Settings** \> **Offboard** \> **Select Operating System to start the offboarding Process**
 
-2. Remove permissions for the third-party solution in your Azure AD tenant.
-   1. Sign in to the [Azure portal](https://portal.azure.com).
-   2. Select **Azure Active Directory > Enterprise Applications**.
-   3. Select the application you'd like to offboard.
-   4. Select the **Delete** button.
+You can also Offboard non-windows devices by disabling the third party integration.Enable coverage for devices running non-Windows platforms by [integrating third-party solutions](https://securitycenter.windows.com/interoperability/partners). 
```

Comment:
```suggestion
You can also Offboard non-windows devices by disabling the third-party integration. Enable coverage for devices running non-Windows platforms by [integrating third-party solutions](https://securitycenter.windows.com/interoperability/partners). 
```

Reply:
This URL is not publicly accessible, please provide a public URL

Reply:
@msbemba, could you follow up, please?